config/locales/dashboard.en.yml

```diff
@@ -223,6 +224,9 @@ en:
       documents:
         header: "Documents"
         add_document: "Add Document"
+        view_study_details: "View Study Details"
```

Comment:
This value is pulled from `t('protocols.view_details.button')` in the application and needs a `protocol_type` parameter.

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/protocols/_summary.html.haml
https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/views/dashboard/protocols/_summary.html.haml